src/components/QuizResults.tsx

```diff
@@ -71,14 +79,21 @@ export const QuizResults = () => {
                     display: "flex",
                     justifyContent: "center"
                 }}>
+
                     <Link href="https://www.sas.com/en_us/curiosity/at-work.html">Learn more about how curiosity is valued in the workplace.</Link>
                 </Box>
                 <Box sx={{
                     mt: 4,
                     display: "flex",
                     justifyContent: "space-between"
                 }}>
-                    <Link href={"/curious"}>Retake Quiz</Link>
+
+                    {/*if loading is true, show loading circle, else show Retake Quiz link*/}
+                    {loading
+                    ? <CircularProgress/>
+                    : <Link onClick={loadingOnClick} href="/curious">Retake Quiz</Link>
+                    }
+
```

Comment:
Do we really need to make this show as loading? I didn't really see any lag here, though maybe there is in certain cases. If we do want to keep it, maybe a better option would be one of these: https://mui.com/material-ui/react-button/#loading-button that way it looks the same.

Reply:
I have had a few times where i've seen lag. but i can look into the loading button, i like that

Reply:
Is loading ever being set to true now? 

I think we should still use Link if we can to represent links, as Buttons are meant for actions that stay on the page. Maybe I was confused by how this was represented.

Reply:
I think that was my confusion on the button, I should have clarified. I'll change it back link but now should have the loading button instead of circular progress